hooks/boot/10-reset_data.rb

```diff
@@ -1,8 +1,7 @@
-# Add reset option
-app_option(
-  '--reset',
+add_option(
+  '--reset-data',
   :flag,
-  'This option will drop the Katello database and clear all subsequent backend data stores. ' +
+  'This option will drop all databases for Foreman and, when present, Katello. ' +
```

Comment:
This opens up to adding in other plugins if they have some sort of data. I tend to prefer just mentioning Foreman and referencing backend systems as a general thing that will get their data cleared.